docs/attach-setup.asciidoc

```diff
@@ -0,0 +1,193 @@
+[[attach-setup]]
+== Setup the agent with the attacher
+
+You can use the attacher as a standalone application on the command line or include it in your application to programmatically attach to the current JVM.
+
+This installation method does not require you to alter the configuration of your application server and can be used to conveniently instrument all JVMs on a particular host.
+
+NOTE: This installation method is experimental.
+
+[float]
+[[attach-setup-cli]]
+=== Use the attacher as a command line application
+
+The `apm-agent-attach.jar` is a small Java program which attaches the Elastic APM Java agent to a specific JVM or to all JVMs of the same host it runs on.
+This requires that the OS user of the attach is the same as the OS user of the applications to be monitored (the JVMs to attach to).
```

Comment:
This paragraph was a little confusing to me, but it might just be because I'm not familiar with some of the terminology. I'm wondering if this would be better:

```suggestion
This requires that the OS user of the attach and the OS user of the application(s) to be monitored (the JVMs to attach to) are the same.
```

Reply:
That sentence is indeed confusing. I was trying to say the OS user used to execute the apm-agent-attach.jar and the JVMs the agent should be attached to have to match.